modin/core/dataframe/pandas/dataframe/utils.py

```diff
@@ -156,8 +157,8 @@ def pick_samples_for_quantiles(
 
 
 def pick_pivots_from_samples_for_sort(
-    df: PandasDataframe,
-    samples: np.ndarray,
+    df: "PandasDataframe",
+    samples: "list[np.ndarray]",
```

Comment:
```suggestion
    samples: list[np.ndarray],
```
to be consistent with code down below

Reply:
If we specify the type of element, it needs to be in quotes, otherwise python fails when we try and load the file!

Reply:
(since list is not subscriptable)